services/espnet-tts-fr/src/app.py

```diff
@@ -51,18 +51,26 @@ def isfloat(num):
 
 def processSegment(s):
     if s.isnumeric() or isfloat(s):
-        logger.debug(f"Case: '{s}' is numeric or float.")
-        return num2words(s, lang='fr')
+        logger.debug("Case: numeric or float.")
+        s = num2words(s, lang='fr')
     elif "," in s:
-        logger.debug(f"Case: '{s}' has , as separator")
+        logger.debug("Case: has a , as separator")
         tempns = s.replace(",", ".")
         if isfloat(tempns):
-            return num2words(float(tempns), lang='fr')
+            s = num2words(float(tempns), lang='fr')
     elif "-" in s:
-        logger.debug("case 3: -")
+        logger.debug("Case: has a - as separator")
```

Comment:
Hmm, this would likely fail for cases where two separators are used. E.g., "39,5-40,0". We could probably do something with regexps with the - separator as well. Get the fields we need excluding text with `([\d,.]+)-([\d,.]+)`, replace with `de $1 à $2` and then apply another find and replace on $1 and $2 to replace occurrences of `,` with `.`. Does this make sense?

Reply:
Yes, it does. I will try them out.